cirq-core/cirq/devices/named_topologies_test.py

```diff
@@ -85,6 +85,10 @@ def test_line_topology():
     assert LineTopology(2).n_nodes == 2
     assert LineTopology(2).graph.number_of_nodes() == 2
 
+    mapping = topo.nodes_to_linequbits(offset=3)
```

Comment:
Final thing - let's verify that mapping keys are as desired

```suggestion
    mapping = topo.nodes_to_linequbits(offset=3)
    assert sorted(mapping.keys()) == list(range(n))
```